### ISSUE TITLE
Handle UK and GB country codes as same country

### ISSUE THREAD

**Issue by KarlisJ** [Automattic/woocommerce-payments#7888]
### Description

Investigate the possibility of handling both the UK and GB as the same country, as both of these are valid country codes. 

### Acceptance criteria

Both the UK and GB can be used to reference the United Kingdom country.

### Additional context

paJDYF-bFG-p2

**Comment by jessy-p** [comment]
This issue impacts International Expansion so assigning to Fusion (based on team responsibilities Pc2DNy-3z-p2) @anu-rock. Assigning as part of Gamma Triage process PcreKM-yM-p2.

**Comment by kalessil** [comment]
More cases: CQ, EU and UK

**Comment by vbelolapotkov** [comment]
This issue is very generic, so I'm closing it based on this comment from @dougaitken : paJDYF-bFG-p2#comment-22902

TLDR; ISO code for UK is `GB` and should be preferred, at least in the code and DB.

If there are specific places in UI or product where UK is preferred, let's create dedicated issues for them.
